src/routers/handlers/enter_your_details/index.ts

```diff
@@ -0,0 +1,96 @@
+import { BaseViewData, GenericHandler, Redirect, ViewModel } from "./../generic";
+import { Request, Response } from "express";
+import { logger } from "../../../utils/logger";
+import { type Address } from "private-api-sdk-node/src/services/presenter-account/types";
+import { PrefixedUrls, Countries } from "../../../constants";
+import { setPresenterAccountDetails, getPresenterAccountDetails, generateUserDetails } from "./../../../utils/session";
+import { FieldValidationError, Result, ValidationError, validationResult } from "express-validator";
+
+interface CountryType {
+    value: string,
+    text: string,
+    selected?: boolean
+}
```

Comment:
Calling this `CountryType` is misleading since it is not defining a country, it is defining an option for a radio button. 
I think this should be renamed to make it more descriptive.